src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/online-unit-form/online-unit-form.component.ts

```diff
@@ -32,32 +33,33 @@ function urlValidator(control: AbstractControl) {
     selector: 'jhi-online-unit-form',
     templateUrl: './online-unit-form.component.html',
 })
-export class OnlineUnitFormComponent implements OnInit, OnChanges {
-    @Input()
-    formData: OnlineUnitFormData;
-    @Input()
-    isEditMode = false;
+export class OnlineUnitFormComponent implements OnChanges {
+    protected readonly faArrowLeft = faArrowLeft;
+    protected readonly faTimes = faTimes;
 
-    @Output()
-    formSubmitted: EventEmitter<OnlineUnitFormData> = new EventEmitter<OnlineUnitFormData>();
-    form: FormGroup;
+    @Input() formData: OnlineUnitFormData;
+    @Input() isEditMode = false;
 
-    @Input()
-    hasCancelButton: boolean;
-    @Output()
-    onCancel: EventEmitter<any> = new EventEmitter<any>();
+    @Output() formSubmitted: EventEmitter<OnlineUnitFormData> = new EventEmitter<OnlineUnitFormData>();
 
-    faTimes = faTimes;
+    @Input() hasCancelButton: boolean;
+    @Output() onCancel: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
The diff is rather hard to read here, the Inputs, Outpus and constants are only reordered